util/storage.js

```diff
@@ -148,6 +150,21 @@ export async function getDirSize(dir) {
   return size;
 }
 
+export async function getDiskUsage(path="/") {
+  const exec = util.promisify(child_process.exec);
+  const result = await exec(`df ${path}`);
```

Comment:
I wonder if this is the best way to check disk usage... i guess maybe it is, instead of using some node library for it?
I suppose if it works well, can go with this for now..

Reply:
I looked for node libraries first (like https://www.npmjs.com/package/check-disk-space and https://www.npmjs.com/package/diskusage) but they all seemed to be wrappers for OS tools like `df` or its Windows equivalent so I figured I'd spare us another dependency and just implement it myself.